sympy/tensor/array/tests/test_ndim_array.py

```diff
@@ -0,0 +1,15 @@
+from sympy import Array
+
+
+def test_array_negative_indices():
+ test_array = Array([[1,2,3,4,5],[6,7,8,9,10]])
```

Comment:
We use the standardized 4 spaces indentation in SymPy.

By the way, maybe this should be put into the other test files? I don't see the point of creating a new file.

Reply:
@sidhantnagpal felt tests were to be in new file.So, new file was added. @Upabjojr , could you please help me in the build failure, for there is failure in first assertion. test_array[:,-1] is [10,5]. However the assertion for the first one is failing, not able to figure out the reason.

Reply:
did you debug?

Reply:
I tested in my local machine, The assertion is "True" for the line to which build is marking as failure.